azext_iot/tests/iothub/test_iot_ext_int.py

```diff
@@ -32,10 +33,28 @@
 
 CWD = os.path.dirname(os.path.abspath(__file__))
 
+test_managed_name = "TESTMANAGEDIDENTITY"
+
 
 class TestIoTStorage(IoTLiveScenarioTest):
     def __init__(self, test_case):
         super(TestIoTStorage, self).__init__(test_case, LIVE_HUB, LIVE_RG)
+        self.managed_identity = None
+
+    def get_managed_identity(self):
+        # Check if there is a managed identity already
+        if self.managed_identity:
+            return self.managed_identity
+
+        # Create managed identity
+        result = self.cmd(
+            "identity create -n {} -g {}".format(
```

Comment:
Is this a create or update, or an idempotent operation i.e. what happens if an identity with the same name exists

Reply:
We should consider clean up of the identity after its usage in the test as well.